ocp_addons_operators_cli/utils/addons_utils.py

```diff
@@ -120,6 +124,15 @@ def assert_addons_user_input(addons, brew_token):
         assert_missing_managed_odh_brew_token(addons=addons, brew_token=brew_token)
 
 
+def store_kubeconfig_from_cluster_name(cluster_name, kubeconfig_dict):
```

Comment:
cluster-object has `kubeconfig`
pass the object and use `cluster.name` and `cluster.kubeconfig`

Reply:
this is not resolved

Reply:
updated the function